src/libraries/System.IO.FileSystem/src/System/IO/FileStatus.Unix.cs

```diff
@@ -26,6 +26,8 @@ internal struct FileStatus
         // Exists as of the last refresh
         private bool _exists;
 
+        internal bool isBrowser { get; private set; }
```

Comment:
Is this needed?

Reply:
Oops not it’s not. I will take that out.